playback/src/gstreamer_backend.rs

```diff
@@ -64,6 +64,7 @@ pub struct GStreamer {
     pub position: Arc<Mutex<i64>>,
     pub duration: Arc<Mutex<i64>>,
     pub radio_title: Arc<Mutex<String>>,
+    _bus_watch_guard: BusWatchGuard
```

Comment:
i had to add somewhere to store the new bus watch guard, otherwise the watch added would be immediately dropped (and so removed from the bus again), or at the end of the function - unlike earlier versions where it did not get removed on drop